common/v2/features/SendAssets/components/TransactionReceipt.tsx

```diff
@@ -89,6 +96,12 @@ export default function TransactionReceipt({ txReceipt, txConfig }: IStepCompone
   const senderAccountLabel = senderContact ? senderContact.label : 'Unknown Account';
 
   const localTimestamp = new Date(Math.floor(timestamp * 1000)).toLocaleString();
+  const assetAmount = txReceipt.amount || txConfig.amount;
+  const assetTicker = txReceipt.asset
+    ? txReceipt.asset.ticker
+    : txConfig.asset
+    ? txConfig.asset.ticker
+    : 'ETH';
```

Comment:
Nested ternaries are often a code smell.
I don't understand why the view accepts values from either Receipt or Config. It should have a single source of truth.

Just for speed sake please indent the nested ternary.